src/ui/ViewModels/SchoolViewModel.cs

```diff
@@ -8,6 +8,7 @@ public class SiteViewModel
         public string Address { get; set; }
         public string Code { get; set; }
         public string Status { get; set; }
+        public string Publish { get; set; }
```

Comment:
What kind of data lives in this string? Perhaps an xmldoc explaining that would be useful